web/content/docs/tools/getting-started/first_steps/index.pandoc

```diff
@@ -0,0 +1,26 @@
++++
+date = "2019-03-14T11:00:13+01:00"
+title = "First Steps"
+author = "Joerg Buchwald"
+
+[menu]
+  [menu.tools]
+    parent = "getting-started"
++++
+
+## Frist Steps
+
+This section describes a possible general pre-processing workflow using some of the tools provided by OGS.
+
+To set up a model domain, the project file `*.prj` requires additional input files for the mesh and geometry to apply the boundary conditions accordingly.
```

Comment:
comma missing:
```
mesh, and geometry to ...
```

Reply:
I don't think so. Maybe without 'the' in front of 'mesh'.

Reply:
Sorry, then I didn't understand what you were saying. Just me, I guess. The sentence is maybe too long and tries to cover several things at once: the meshes for the project, and then other meshes/geometries for BCs.
Never mind the comma.